### DISCUSSION TITLE
Great developers, how can I change the program from installing to /opt to something else, such as /etc?

### DISCUSSION THREAD

**Discussion by MortalM** [Entware/Entware#1109] [Q&A]
my device is xiaomi-AX3000T and redmi-AX6000, it's a route system ,program can't install on  /opt , becasue / is read only system, can we chose the install /opt to other place , e.g.   to /etc 
Great developer please help me!   This question has been bothering me for half a year now
![image](https://github.com/user-attachments/assets/5ac792a7-8d4e-4d9e-82d7-fc91c5a16170)
If the issue is resolved, I will proactively close this session

**Comment by MortalM** [comment]
@ryzhovau @themiron @MercuryV @neheb

**Reply by MortalM** [reply]
I re-edited the alternative.sh file, changed /opt to /tmp/opt, and then after installing it, it reported an error, because the opkg command solidified the configuration file address to /opt/etc/opkg.conf, how can I change the configuration file address obtained by the opkg file to /tmp/opt/etc/opkg.conf?
![image](https://github.com/user-attachments/assets/e27c90b8-77ea-43dc-8efd-f3990617b2d2)

**Comment by jethroso** [comment]
Because your router path is /tmp/opt/etc, the actual point is still /opt/etc. So, you need to insert a USB flash drive, which can be installed on the USB flash drive.
